CODEOWNERS

```diff
@@ -0,0 +1,59 @@
+# See also:
+# - https://github.com/blog/2392-introducing-code-owners
+# - https://help.github.com/articles/about-codeowners/
+
+# Each line is a file pattern followed by one or more owners (sorted alphabetically).
+# Please feel free to add yourself to a module or create a new mapping.
+# Ideally each module should have two or more owners.
+
+# Code owners are automatically requested for review
+# when someone opens a pull request that modifies code that they own.
+# Later matches take precedence.
+
+src/checkedint.d @redstar @andralex @JackStouffer
+
+src/core/atomic.d @WalterBright
+src/core/attribute.d @jaoc-carlborg
+src/core/bitop.d @schveiguy @tsbockman @mathias-lang-sociomantic @Geod24
```

Comment:
@mathias-lang-sociomantic - did you ask for yourself added twice here? (Have I given away too much information by suggesting you have a second identity?)

Reply:
Yes he did: 

> Could you list my personal account (@Geod24 ) along my Sociomantic one ? Thanks.

So going undercover now would require a third account ;-)